docs/rbac-setup.md

```diff
@@ -0,0 +1,13 @@
+
+# RBAC Setup 
+
+If rbac is enabled, allow the service catalog to grant access for it's
+service token.
+
+## Install our rbac rules
+
+See the [script in contrib](../contrib/rbac-setup.sh). See the
```

Comment:
Should they run the script in contrib to install the RBAC rules? If so, can you add that little piece to the docs